corehq/apps/linked_domain/applications.py

```diff
@@ -92,15 +92,20 @@ def get_linked_apps_for_domain(domain):
     return linked_apps
 
 
+def _create_linked_app(domain, app_name):
```

Comment:
I'm ambivalent about this as a helper method. Looking at the final diff, there's both a `create_linked_app` and a `_create_linked_app` which I don't think makes it obvious what the different purposes of the methods are.

Plus if you moved the import to the top level, this could be a one-liner, which makes me feel like it isn't substantial enough to be a method. I see the final version also has a `save` call, but I think it's debatable which code should have responsibility for saving.

Reply:
Totally agree. I can consolidate that into one method. Worth noting the import is done there to avoid a circular import error.